### PR TITLE
update: treeNodeRenderer method added to treePickerGrid

### PR DESCRIPTION
**Pull request by sharmajyo**
Created a treeNodeRenderer function in treePickerGrid so that we can customise the value in cell instead of showing only label.

screen shot 2016-08-23 at 11 56 19 am

### CODE REVIEW

**Comment by vinteo** [comment]
LGTM

File: src/components/adslotUi/TreePickerNodeComponent.jsx [hunk 1]
```diff
@@ -48,6 +49,14 @@ const TreePickerNodeComponent = ({
     { onClick: expandNodeBound } :
     {};
 
+  const renderLabel = (node) => {
```
Comment (OndrejCholeva) [inline src/components/adslotUi/TreePickerNodeComponent.jsx]: Why you can't make `treeNodeRenderer: PropTypes.func.isRequired` with `treeNodeRenderer: (node) => node.label,` as a default? (see `valueFormatter`)

Comment (OndrejCholeva) [inline src/components/adslotUi/TreePickerNodeComponent.jsx]: and actually rename it all to `labelFormatter` or `labelRenderer` as it describes a bit better what it is doing. (the `tree` word seems extra, as it is just a prop - we don't name props by component)

Comment (sharmajyo) [inline src/components/adslotUi/TreePickerNodeComponent.jsx]: its not technically a required property for this component so there is no good reason for forcefully making it required, I suppose...
Also in various open source react components we are using (like react-select) such custom rendering properties are always optional.

Comment (OndrejCholeva) [inline src/components/adslotUi/TreePickerNodeComponent.jsx]: Default should work even without required, I just don't see a reason for reinventing existing functionality.

**Comment by pphminions** [comment]
LGTM 🍒

**Comment by sharmajyo** [comment]
@OndrejCholeva review comments added.

**Comment by vinteo** [comment]
Merging...

**Pull request merged**